dracut/03flatcar-network/afterburn-network-kargs.service

```
@@ -0,0 +1,22 @@
+[Unit]
+Description=Afterburn Initrd Setup Network Kernel Arguments
+Documentation=https://coreos.github.io/afterburn/usage/initrd-network-cmdline/
+
+# This service may produce additional kargs fragments,
+# which are then consumed by dracut-cmdline(8).
+DefaultDependencies=no
+Before=dracut-cmdline.service systemd-networkd.service
+PartOf=systemd-networkd.service
+
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+[Service]
+Environment=AFTERBURN_OPT_PROVIDER=--cmdline
+Environment=AFTERBURN_NETWORK_KARGS_DEFAULT='ip=dhcp,dhcp6'
```

```suggestion
Environment=AFTERBURN_NETWORK_KARGS_DEFAULT=''
```
If that works we can avoid that the `/etc/systemd/network/10-dracut-cmdline-….network` file takes precedence over our default config file.

---

yeah - will try that once the feature stabilized. I'm a bit afraid of hitting this: https://github.com/coreos/afterburn/blob/main/src/cli/exp.rs#L46

---

Reading the code it looks like an empty string is ok. The question it maybe more about how the quoting in the systemd unit here is done, which I think is correct already (cf. https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines).